dev-libs/libclc/libclc-0.2.0_pre20180610.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_4 python3_5 python3_6 )
+
+EGIT_REPO_URI="http://llvm.org/git/${PN}.git
+	https://github.com/llvm-mirror/${PN}.git"
+EGIT_COMMIT="96627043299f52bf08f6a68aa6d00b8037a0a2b0"
+
+if [[ ${PV} = 9999* ]]; then
+	GIT_ECLASS="git-r3"
+	EXPERIMENTAL="true"
+else
+	GIT_ECLASS="vcs-snapshot"
```

Comment:
Can I ask you to kill vcs-snapshot while at it? The `S` seems to be entirely predictable here.

Reply:
I think we need to keep this one. The github tar.gz unpacks to libclc-$GIT_HASH so we need to rename the directory, which is exactly what the eclass does.

Reply:
You need to know the hash anyway, so just update `S` appropriately.

Reply:
ah, now I get it. My ebuild-fu is weak. Thanks.